fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImpl.java

```diff
@@ -356,4 +326,105 @@ public Integer getTimeout() {
     public DruidServiceConfig getServiceConfig() {
         return serviceConfig;
     }
+
+    protected Meter getHttpErrorMeter() {
+        return httpErrorMeter;
+    }
+
+    protected Meter getExceptionMeter() {
+        return exceptionMeter;
+    }
+
+    protected DruidServiceConfig getDruidServiceConfig() {
+        return serviceConfig;
+    }
+
+    /**
+     * Log request using RequestLog.
```

Comment:
It's also worth noting that this isn't logging the request, but it's logging the completed response. If it was logging the request, it would not need to be triggering with pieces of the response. And if it was for any response, it should handle the failure cases as well.